pkg/bot/slack.go

```diff
@@ -4,27 +4,51 @@ import (
 	"context"
 	"fmt"
 	"strings"
+	"sync"
 
 	"github.com/sirupsen/logrus"
 	"github.com/slack-go/slack"
 
 	"github.com/kubeshop/botkube/internal/analytics"
 	"github.com/kubeshop/botkube/pkg/config"
+	"github.com/kubeshop/botkube/pkg/events"
+	format2 "github.com/kubeshop/botkube/pkg/format"
+	"github.com/kubeshop/botkube/pkg/multierror"
 )
 
-// SlackBot listens for user's message, execute commands and sends back the response
-type SlackBot struct {
+// TODO: Refactor:
+// 	- handle and send methods from `slackMessage` should be defined on Bot level,
+//  - split to multiple files in a separate package,
+//  - review all the methods and see if they can be simplified.
+
+var _ Bot = &Slack{}
+
+const sendFailureMessageFmt = "Unable to send message to ChannelName `%s`: `%s`\n```add Botkube app to the ChannelName %s\nMissed events follows below:```"
+const channelNotFoundCode = "channel_not_found"
```

Comment:
```suggestion
const (
	sendFailureMessageFmt = "Unable to send message to channel `%s`: `%s`\n```add Botkube app to the channel %s\nMissed events follows below:```"
	channelNotFoundCode   = "channel_not_found"
)
```